### ISSUE TITLE
Broken on MW 1.27, doesn't load page, then when fixed, action links broken, user links broken

### ISSUE THREAD

**Issue by HopperMCS** [dantman/monaco-port#15]
Not quite sure what you meant in your response to the pull request (that was meant for James Haley, tried to get ahold of him but I was on vacation so all I had was an iPhone and the glitch slipped past me) about changes made "in April." Your repo says the last commit was made in September 2014.

This doesn't quite work right out of the box like I was hoping it would, but I don't believe that's enough context. Yes, this is yours, I got the right one, dantman. See, on the one I was doing fixing for before, even AFTER I fixed some of these issues (even though you'll probably tell me I broke more than fixed), other weird things happened that caused me to do some unorthodox things. For example, the action links displayed some weird "attributes," and the login worked, however once logged in, the user links would break. I don't know how to fix that, but if you actually tried my changes on a localhost install, you could see what I did as a workaround. If you do fix this up, I would gladly get all my stuff out of the code, and use them as extensions (yes, the ones that go in the /extensions folder on a base MediaWiki install) to yours.

PS I don't do object-oriented PHP. Just not my thing I guess. I'm more than capable of doing WordPress themes, but MediaWiki is too much of a mess for me to understand. WP has documentation, but MediaWiki is sometimes just not all the way there. 

![alt-text](http://i.imgur.com/voWhKYQ.png Inserted error)

Cheers!

**Comment by streamside7** [comment]
Same here. Issues with 1.28

**Comment by HopperMCS** [comment]
Yup, same issue I faced with a fork of this port by James Haley way back in July. Dan has been less than cordial about those of us who still use this, though (which baffles me, but I guess I get it if he doesn't want to maintain it)

As a good colleague once told me, "In programming, you will never be finished with a project." Just some food for thought.

**Comment by HopperMCS** [comment]
@streamside7 I doubt he's gonna fix it. I filed this back last year. Forgot I filed it completely. I do object-oriented programming languages, but PHP ain't built for that...it's kind of bolted on, clunky, and less than sexy. Props to @dantman for his work back in the day, but looks like he fell off.

I still like the appearance of Monaco, but based on where the Net is going, Monaco hasn't a way to fit in. IDK, but I feel a cleanroom design would be better and easier to maintain, because obviously when porting this it involved tedious, numerous runs through hell and back. Just look at that commit log :p

That may be a summer project for me: Cleanroom Monaco, and make it mobile friendly for God's sake.

**Comment by dantman** [comment]
> Props to @dantman for his work back in the day, but looks like he fell off.

Yup, I have too many other projects, monaco-port is a fork of a poorly written Wikia skin, and the only project I had left using it has been shut down.

**Comment by HopperMCS** [comment]
@dantman I plan to completely re-write, using none of the old code in this repo. I'll be re-theming Bootstrap and then attempting to shove it into a MediaWiki skin. Why Bootstrap? Mobile-first + re-usable components = designer-friendly.
